charts/fission-core/values.yaml

```diff
@@ -269,3 +269,6 @@ mqt_keda:
     gcp_pub_sub:
       image: fission/keda-gcp-pubsub-http-connector
       tag: v0.2
+    redis:
+      image: blackfly19/keda-redislist-http-connector
+      tag: v5
```

Comment:
Since this would be first release for redis-http-connector, we should have version v0.1